data/lt/ltextras/collectible/milestone/philza_baby_zombie.json

```diff
@@ -0,0 +1,71 @@
+{
+  "components": {
+    "minecraft:lore": [
+      "{\"translate\":\"lt.collectible.baby_zombie.description\",\"color\":\"gray\",\"italic\":true}",
+      "{\"text\":\" \"}",
+      "{\"translate\":\"lt.collectible.unicode.reward\",\"color\":\"white\",\"italic\":false}",
+      "{\"text\":\" \"}",
+      "{\"translate\":\"item.modifiers.head\",\"color\":\"gray\",\"italic\":false}",
+      "{\"translate\":\"lt.collectible.disguise.tooltip.subtract_player\",\"color\":\"blue\",\"italic\":false}",
+      "[{\"text\":\" + \",\"color\":\"blue\",\"italic\":false},{\"translate\":\"lt.collectible.baby_zombie.name\",\"color\":\"blue\",\"italic\":false}]"
+    ],
+    "minecraft:enchantments": {
+      "show_in_tooltip": 0,
+      "levels": {}
+    },
+    "ltminigames:disguise": {
+      "entity": {
+        "tag": {
+          "IsBaby": 1,
+          "ArmorItems": [
```

Comment:
Ohh, we aren't data-fixing inside of these undescribed tags :sweat_smile: 
We could probably hack it into the schema for running the upgrade, or manually fix it up now